go.mod

```diff
@@ -78,3 +78,5 @@ replace (
 	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.19.3
 	k8s.io/sample-controller => k8s.io/sample-controller v0.19.3
 )
+
+replace github.com/kube-object-storage/lib-bucket-provisioner => github.com/BlaineEXE/lib-bucket-provisioner v0.0.0-20201124191107-00153dd47f37
```

Comment:
@BlaineEXE : Is this temporary until [PR](https://github.com/kube-object-storage/lib-bucket-provisioner/pull/198) merges?

Reply:
Yes. It's gone now.